src/sentry/runner/__init__.py

```diff
@@ -1,11 +1,16 @@
+import logging
 import os
 import click
 import sys
 import sentry
 import datetime
+import sentry_sdk
 from sentry.utils.imports import import_string
 from sentry.utils.compat import map
 
+logging.basicConfig(format="%(asctime)s [%(levelname)s] %(message)s", level=logging.INFO)
```

Comment:
would only run this based on `SENTRY_DEVENV_DSN` to avoid potential impact on prod